fix(rewrite): avoid wildcards on destination topic on publish (#11004)
emqx/emqx: changes/ce/fix-11004.en.md

> Avoid wildcards on destination topic for rewrite rules.

Reviewer: ```suggestion
Do not allow wildcards for destination topic in rewrite rules.
```

Author: Ack
